src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/mapper/InductionMigrationMapper.kt

```diff
@@ -0,0 +1,62 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration.mapper
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration.jpa.entity.InductionMigrationEntity
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration.resource.model.CiagInductionResponse
+import java.time.ZoneOffset
+import java.util.UUID
+
+@Component
+class InductionMigrationMapper(
+  private val workOnReleaseMapper: WorkOnReleaseMigrationMapper,
+  private val qualificationsMapper: PreviousQualificationsMigrationMapper,
+  private val trainingMapper: PreviousTrainingMigrationMapper,
+  private val workExperiencesMapper: PreviousWorkExperiencesMigrationMapper,
+  private val inPrisonInterestsMapper: InPrisonInterestsMigrationMapper,
+  private val skillsAndInterestsMapper: SkillsAndInterestsMigrationMapper,
+  private val workInterestsMapper: FutureWorkInterestsMigrationMapper,
+) {
+  fun toInductionMigrationEntity(ciagInduction: CiagInductionResponse): InductionMigrationEntity =
+    with(ciagInduction) {
+      // prisonId shouldn't be null, but we've seen bugs here on the CIAG side recently.
+      // We can default it for now, so long as the service is still only rolled out to Lancaster Farms!
+      val prisonId = ciagInduction.prisonId ?: "LFI"
```

Comment:
I'm defaulting this for now, but I might need to change this. As I've said in the comments, it shouldn't be null anyway, but it's not mandatory in their API spec.